### PR TITLE
Support including images in object type flow inputs

### PR DESCRIPTION
**Pull request by lumoslnt**
# Description

We only support including images in list type flow inputs before, in this PR, we support including images in object type flow inputs.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by thy09 — APPROVED** [review]

File: src/promptflow/promptflow/_utils/multimedia_utils.py [hunk 1]
```diff
@@ -193,3 +195,25 @@ def recursive_process(value: Any, process_funcs: dict[type, Callable] = None) ->
...
+    updated_inputs = dict(line_inputs or {})
+    for key, value in inputs.items():
+        if value.type == ValueType.IMAGE:
+            updated_inputs[key] = create_image(updated_inputs[key], base_dir)
+        elif value.type == ValueType.LIST or value.type == ValueType.OBJECT:
+            updated_inputs[key] = load_multimedia_data_recursively(updated_inputs[key])
+    return updated_inputs
+
+
+def load_multimedia_data_recursively(value: Any):
```
Comment (huaiyan) [inline src/promptflow/promptflow/_utils/multimedia_utils.py:210]:
```suggestion
def _load_multimedia_data_recursively(value: Any):
```
Will be a internal method?

File: src/promptflow/promptflow/executor/flow_executor.py [hunk 2]
```diff
@@ -697,7 +696,7 @@ def exec_line(
         """
         self._node_concurrency = node_concurrency
         inputs_with_default_value = FlowExecutor._apply_default_value_for_input(self._flow.inputs, inputs)
-        inputs = self._process_images_from_inputs(self._flow.inputs, inputs_with_default_value)
+        inputs = load_multimedia_data(self._flow.inputs, inputs_with_default_value, self._working_dir)
```
Comment (huaiyan) [inline src/promptflow/promptflow/executor/flow_executor.py:699]: What if customer's dict with image value is passed in as node.input?

**Review by huaiyan — APPROVED** [review]

File: src/promptflow/promptflow/executor/flow_executor.py (hunk 2 above)
Comment (lumoslnt) [reply src/promptflow/promptflow/executor/flow_executor.py:699]: Offline synced, will fix in new PR.

File: src/promptflow/promptflow/_utils/multimedia_utils.py (hunk 1 above)
Comment (lumoslnt) [reply src/promptflow/promptflow/_utils/multimedia_utils.py:210]: Offline synced, will fix in new PR.

File: src/promptflow/promptflow/_utils/multimedia_utils.py [hunk 3]
```diff
@@ -193,3 +195,25 @@ def recursive_process(value: Any, process_funcs: dict[type, Callable] = None) ->
     if isinstance(value, dict):
         return {k: recursive_process(v, process_funcs) for k, v in value.items()}
     return value
+
+
+def load_multimedia_data(inputs: dict[str, FlowInputDefinition], line_inputs: dict, base_dir: Path):
```
Comment (Hhhilulu) [inline src/promptflow/promptflow/_utils/multimedia_utils.py:200]: May be can remove base_dir.
We should be loading from either absolute path or relative path to current working dir.
In other words, the path should be always readable without modifying.

Comment (lumoslnt) [reply src/promptflow/promptflow/_utils/multimedia_utils.py:200]: 1. Good point, will remove in new PR.
2. It is not a class method, you can call it directly in flow executor.

Comment (Hhhilulu) [reply src/promptflow/promptflow/_utils/multimedia_utils.py:200]: Ok, I see

**Pull request merged**